virttest/utils_disk.py

```diff
@@ -49,35 +49,56 @@ def copytree(src, dst, overwrite=True, ignore=''):
             shutil.copy(src_file, dst_dir)
 
 
-def is_mount(src, dst):
+def is_mount(src, dst, verbose=False, session=None):
```

Comment:
IMHO, we should add `fstype` and `option` here.

Reply:
@luckyh Updated, please help review, thanks.